cloud/scope/cluster.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scope
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"k8s.io/klog/klogr"
+	infrav1 "sigs.k8s.io/cluster-api-provider-digitalocean/api/v1alpha2"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha2"
+	"sigs.k8s.io/cluster-api/util/patch"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
I would prefer if we would keep the old imports rule, i.e.:
* Go SDK
* External packages, e.g. `logr`
* Project packages, i.e. `sigs.k8s.io/cluster-api-provider-digitalocean`
* Other Kubernetes packages

If there are any objections to that, I'd at least split project packages and all other packages. This is done by the AWS provider.

Reply:
i just reorganized the import packages. ptal @xmudrii @cpanato

Reply:
Looks good to me, thanks!